Assets/Mirage/Runtime/NetworkIdentity.cs

```diff
@@ -678,10 +681,14 @@ void OnDeserialize(NetworkBehaviour comp, NetworkReader reader, bool initialStat
             byte barrierData = reader.ReadByte();
             if (barrierData != Barrier)
             {
-                throw new DeserializeFailedException($"Deserialize not aligned for object={name} netId={NetId} component={comp.GetType()} sceneId={SceneId:X}. Possible Reasons:\n" +
-                    $"  * Do {comp.GetType()}'s OnSerialize and OnDeserialize calls write the same amount of data? \n" +
-                    $"  * Are the server and client the exact same project?\n" +
-                    $"  * Maybe this OnDeserialize call was meant for another GameObject? The sceneIds can easily get out of sync if the Hierarchy was modified only in the client OR the server. Try rebuilding both.\n\n");
+                // Coburn: something something not aligned...? "Deserialize not aligned [...]" ... also could we show the amount of data in our buffer?
```

Comment:
It would be hard to show the data in the buffer because we might be in the middle of the byte (if first component ends at bit 2, next start at bit 3) so we'd have to shift the buffer (possible but would require a fair amount of code).

Reply:
We also dont know the length of the data, so we'd have to dump the rest of the buffer to the log message.

Reply:
Alright, thanks for letting me about that. And yeah, if we dumped the rest of the buffer it would potentially be really messy (or if it was a large buffer, then we'd have a lot of console/debug message spam).